client/src/pages/Home/Home.tsx

```diff
@@ -2,33 +2,36 @@
 import React, { useState, useContext, useEffect} from 'react';
 import SearchBar from 'components/Shared/SearchBar/SearchBar';
 import { useGetInteractionsByGenes, useGetInteractionsByDrugs } from 'hooks/queries/useGetInteractions';
-import { useNavigate } from 'react-router-dom';
+import { createSearchParams, useNavigate } from 'react-router-dom';
 import { GlobalClientContext } from 'stores/Global/GlobalClient';
 import { ActionTypes } from 'stores/Global/reducers';
 
 import { queryClient } from 'providers/app';
 
 // styles
-import { Button, Switch } from 'antd';
-import SunIcon from 'components/Shared/SVG/SunIcon';
-import MoonIcon from 'components/Shared/SVG/MoonIcon';
+import { Button } from 'antd';
+// todo: introduce dark mode back later
+// import SunIcon from 'components/Shared/SVG/SunIcon';
+// import MoonIcon from 'components/Shared/SVG/MoonIcon';
 import './Home.scss';
 
 export const Home: React.FC = () => {
 
   const {state, dispatch} = useContext(GlobalClientContext);
+  const navigate = useNavigate();
 
   const handleSubmit = async () => {
     dispatch({type: ActionTypes.ContentPage})
     if (state.interactionMode === 'categories') {
       navigate('/categories');
     } else {
-      navigate('/results');
+      navigate({
+        pathname: '/results',
+        search: `?${createSearchParams({searchTerms: state.searchTerms.join(',')})}`,
```

Comment:
Gene symbols I believe will be alphanumeric, but concept ids may include `:` characters which would require URL encoding to work correctly here I believe, unless `createSearchParams` is handling that under the hood.

Also looking at the docs, I believe you may be able to omit the leading `?`

Reply:
ah yeah, was going to ask about this since I'm not sure all of the possibilities for gene/drug names. It appears that `createSearchParams` handles the encoding for us, which is nice!